src/language/parser.js

```diff
@@ -117,6 +117,19 @@ export type ParseOptions = {
    * disables that behavior for performance or testing.
    */
   noLocation?: boolean,
+
+  /**
+   * If enabled, the parser will understand and parse variable definitions
+   * contained in a fragment definition. They'll be represented in the
+   * `variableDefinitions` field of the FragmentDefinitionNode.
```

Comment:
Should we make a point that this is not guaranteed to be supported in the future, and anyone that depends on this may have to migrate away from it at a later date?

Reply:
Agreed - highlight the fact this is experimental and may change or be removed in the future